db/models.js

```diff
@@ -7,3 +7,21 @@ exports.fetchTopics = () => {
     });
 }
 
+exports.fetchArticle = (id) => { 
+    id = parseInt(id)
+    if(!id) {
```

Comment:
I would test this with a 0 as well as your if(!id) will also give you the 400 error if the path is /api/articles/0 but this shouldn't be a 400! This may be helpful: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN